dot/network/host_test.go

```diff
@@ -37,7 +38,8 @@ func TestConnect(t *testing.T) {
 		NoMdns:      true,
 	}
 
-	nodeA, _, _ := createTestService(t, configA)
+	blockState := newMockBlockState(big.NewInt(1))
```

Comment:
can we include making a new mock block state in the `createTestService` helper method instead of having to do this for each test within the network package?

Reply:
sure, I created another method for the custom state then.